source/_components/sensor.teksavvy.markdown

```diff
@@ -0,0 +1,56 @@
+---
+layout: page
+title: "TekSavvy"
+description: "Instructions how to integrate TekSavvy data usage within Home Assistant."
```

Comment:
**Grammar**:
- `Instructions how to integrate` - `Instructions on how to integrate`